src/org/usfirst/frc/team2399/robot/RobotMap.java

```diff
@@ -9,67 +9,80 @@
  * floating around.
  */
 public class RobotMap {
-
-	/**
-	 * Creates instances of subsystems for use in commands
-	 */
 	
 
 	/**
 	 * ADDRESSES, PORTS, AND CHANNELS
-	 */
-
-	/**
 	 * Address for Talons Things on the CAN network have addresses. Things on
 	 * PWM network have ports. TODO: Find correct cantalon once on the robot
+	 * 
 	 */
 	
-	
+	public static final int DRIVETRAIN_RIGHT_TALON_FRONT_ADDRESS = 7;
+	public static final int DRIVETRAIN_LEFT_TALON_FRONT_ADDRESS = 10;
+	public static final int DRIVETRAIN_LEFT_BACK_TALON_ADDRESS = 6;
+	public static final int DRIVETRAIN_RIGHT_BACK_TALON_ADDRESS = 9;
+	public static final int DRIVETRAIN_LEFT_MIDDLE_TALON_ADDRESS = 8;
+	public static final int DRIVETRAIN_RIGHT_MIDDLE_TALON_ADDRESS = 5;
+	public static final int CLIMBER_TALON_ADDRESS = 2;
 
-	public static final int DRIVETRAIN_RIGHT_TALON_FRONT_ADDRESS = 1;
-	public static final int DRIVETRAIN_LEFT_TALON_FRONT_ADDRESS = 4;
-	public static final int DRIVETRAIN_LEFT_BACK_TALON_ADDRESS = 3;
-	public static final int DRIVETRAIN_RIGHT_BACK_TALON_ADDRESS = 2;
-	public static final int CLIMBER_TALON_ADDRESS = 2399;
-
-	/**
-	 * Ports for sensors and joysticks
-	 */
 	public static final int JOYDRIVE_LEFT_STICK_PORT = 0;
 	public static final int JOYDRIVE_RIGHT_STICK_PORT = 1;
 
 	/**
-	 * Forward Constants To ensure positive values of motion are what you
+	 * FORWARD CONSTANTS
+	 * To ensure positive values of motion are what you
 	 * expect.
-	 *
+	 * TODO: Set actual forward constants for encoders
 	 */
 
 	public static final int JOYDRIVE_FORWARD_CONSTANT = -1;
```

Comment:
New rule, which I should've mentioned before: Constants should not end in _CONSTANT. The fact that they're capitalized and in RobotMap tells us what they are; the extra _CONSTANT suffix is redundant. Please rename all appropriate constants.